core/services/feeds/service.go

```diff
@@ -1143,6 +1152,19 @@ func (s *service) getCSAPrivateKey() (privkey []byte, err error) {
 	return keys[0].Raw(), nil
 }
 
+// getWorkflowKey gets the server's Workflow key
+// because we will have only one key, we can get the first key
+func (s *service) getWorkflowKey() (*workflowkey.Key, error) {
+	keys, err := s.workflowKeyStore.GetAll()
+	if err != nil {
+		return nil, err
+	}
+	if len(keys) < 1 {
+		return nil, errors.New("Workflow key does not exist")
+	}
```

Comment:
Not all nodes will have this key, so if you can't fetch it we should just leave it blank.